.github/ISSUE_TEMPLATE/crash-issues.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Crash Report
+about: For issues regarding game crashes or permanent freezes
+---
+
+<!-- After you fill in all information, delete all comments in the issue -->
+
+**What is your problem:** <!-- Name the problem you have encountered -->
+
+**Describe your problem:** <!-- Provide any information you believe could be useful -->
+
+**Screenshots or videos showing encountered issue:** 
+
+**osu!lazer version:** <!-- Provide the version of your osu!lazer, you can find it at the bottom of the screen  -->
+
+**Logs:** <!-- Attach your osu!lazer logs, you can find them under %appdata%\osu\logs -->
```

Comment:
What about mac or linux?

Reply:
i would have to know where logs are located in those OS, so i could add them 🤷‍♂️

Reply:
For mac its in ` ~/.local/share/osu/`, might be the same for linux?